pages/common/csvpy.md

```diff
@@ -0,0 +1,12 @@
+# csvpy
+
+> Loads a CSV file into a Python shell.
+> Included in csvkit.
+
+- Load a CSV file into a _CSVKitReader_ object:
```

Comment:
Is this meant to be markdown formatting? Because github certainly seems to be formatting it as italics.